src/main/kotlin/com/vanniktech/android/apk/size/ApkSizeExtension.kt

```diff
@@ -1,8 +1,8 @@
 package com.vanniktech.android.apk.size
 
-class ApkSizeExtension {
+open class ApkSizeExtension {
```

Comment:
does it need to be open?

Reply:
Throws an exception when creating an extension.

Reply:
```
com.vanniktech.android.apk.size.ApkSizePluginSpec > android - all tasks created FAILED
    org.gradle.api.ProjectConfigurationException: A problem occurred configuring root project 'test'.
        at org.gradle.configuration.project.LifecycleProjectEvaluator.addConfigurationFailure(LifecycleProjectEvaluator.java:94)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:89)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.doConfigure(LifecycleProjectEvaluator.java:70)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.access$100(LifecycleProjectEvaluator.java:34)
        at org.gradle.configuration.project.LifecycleProjectEvaluator$ConfigureProject.run(LifecycleProjectEvaluator.java:110)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:50)
        at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:666)
        at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:135)
        at com.vanniktech.android.apk.size.ApkSizePluginSpec.android - all tasks created(ApkSizePluginSpec.groovy:68)

        Caused by:
        org.gradle.api.GradleException: Could not generate a proxy class for class com.vanniktech.android.apk.size.ApkSizeTask.
            at org.gradle.api.internal.AbstractClassGenerator.generateUnderLock(AbstractClassGenerator.java:223)
            at org.gradle.api.internal.AbstractClassGenerator.generate(AbstractClassGenerator.java:79)
            at org.gradle.api.internal.project.taskfactory.TaskFactory.create(TaskFactory.java:117)
            at org.gradle.api.internal.project.taskfactory.TaskFactory.create(TaskFactory.java:39)
            at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory.create(AnnotationProcessingTaskFactory.java:49)
            at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory.create(AnnotationProcessingTaskFactory.java:30)
            at org.gradle.api.internal.project.taskfactory.DependencyAutoWireTaskFactory.create(DependencyAutoWireTaskFactory.java:44)
            at org.gradle.api.internal.project.taskfactory.DependencyAutoWireTaskFactory.create(DependencyAutoWireTaskFactory.java:27)
            at org.gradle.api.internal.tasks.DefaultTaskContainer$TaskInstantiator.create(DefaultTaskContainer.java:301)
            at org.gradle.api.internal.tasks.DefaultTaskContainer$TaskInstantiator.create(DefaultTaskContainer.java:289)
            at org.gradle.api.internal.tasks.DefaultTaskContainer.create(DefaultTaskContainer.java:116)
            at com.vanniktech.android.apk.size.ApkSizePlugin$applyAndroidProject$1.execute(ApkSizePlugin.kt:36)
            at com.vanniktech.android.apk.size.ApkSizePlugin$applyAndroidProject$1.execute(ApkSizePlugin.kt:10)
            at org.gradle.internal.ImmutableActionSet$SingletonSet.execute(ImmutableActionSet.java:155)
            at org.gradle.internal.MutableActionSet.execute(MutableActionSet.java:35)
            at org.gradle.api.internal.DefaultDomainObjectCollection.doAdd(DefaultDomainObjectCollection.java:204)
            at org.gradle.api.internal.DefaultDomainObjectCollection.add(DefaultDomainObjectCollection.java:198)
            at com.android.build.gradle.AppExtension.addVariant(AppExtension.java:80)
            at com.android.build.gradle.internal.ApiObjectFactory.create(ApiObjectFactory.java:137)
            at com.android.build.gradle.BasePlugin.lambda$createAndroidTasks$5(BasePlugin.java:651)
            at com.android.builder.profile.ThreadRecorder.record(ThreadRecorder.java:81)
            at com.android.build.gradle.BasePlugin.createAndroidTasks(BasePlugin.java:636)
            at com.android.build.gradle.BasePlugin.lambda$null$3(BasePlugin.java:555)
            at com.android.builder.profile.ThreadRecorder.record(ThreadRecorder.java:81)
            at com.android.build.gradle.BasePlugin.lambda$createTasks$4(BasePlugin.java:551)
            at org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:91)
            at org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:80)
            at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:42)
            at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:230)
            at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:149)
            at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:58)
            at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:324)
            at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:234)
            at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:140)
            at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:37)
            at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
            at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:76)
            ... 11 more

            Caused by:
            java.lang.VerifyError: Cannot inherit from final class
                at org.gradle.internal.classloader.ClassLoaderUtils.define(ClassLoaderUtils.java:104)
                at org.gradle.model.internal.asm.AsmClassGenerator.define(AsmClassGenerator.java:58)
                at org.gradle.model.internal.asm.AsmClassGenerator.define(AsmClassGenerator.java:54)
                at org.gradle.api.internal.AsmBackedClassGenerator$ClassBuilderImpl.generate(AsmBackedClassGenerator.java:967)
                at org.gradle.api.internal.AbstractClassGenerator.generateUnderLock(AbstractClassGenerator.java:221)
                ... 47 more
```